modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase

```diff
@@ -312,6 +312,99 @@ definition {
 		}
 	}
 
+	@description = "LPS-144474 - Verify it's possible to deactivate the custom validation"
+	@priority = 4
+	test CanDeactivateValidation {
+		property ci.retries.disabled = "true";
+		property custom.properties = "feature.flag.LPS-187846=true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given a custom object definition and a long text field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 192646",
+				objectName = "CustomObject192646",
+				pluralLabelName = "Custom Objects 192646");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "LongText",
+				fieldLabelName = "Long Text",
+				fieldName = "longText",
+				fieldType = "Clob",
+				isRequired = "false",
+				objectName = "CustomObject192646");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject192646");
+		}
+
+		task ("and Given a groovy script validation set to full validation") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 192646");
+
+			ObjectAdmin.goToValidationsTab();
+
+			ObjectCustomValidation.addObjectValidation(
+				validationName = "Custom Validation",
+				validationType = "Groovy");
+
+			ObjectCustomValidation.gotoCustomValidation(validationName = "Custom Validation");
+
+			ObjectCustomValidation.activeValidation();
+
+			ObjectCustomValidation.gotoTab(tabName = "Conditions");
+
+			ObjectCustomValidation.insertScriptOnValidation(scriptValidation = "invalidFields = (longText == 'Build Incredible Digital Experiences with Liferay DXP')");
+
+			ObjectCustomValidation.addErrorMessage(errorMessage = "This entry is not valid for long text field.");
+
+			PortletEntry.save();
+		}
+
+		task ("When attempting an object entry with incorrect validation") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject192646");
+
+			LexiconEntry.gotoAdd();
+
+			FormViewBuilder.editClobText(fieldLong = "Build Incredible Digital Experiences with Liferay DXP");
+
+			Button.clickSave();
+		}
+
+		task ("Then assert the entry was not added") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject192646");
+
+			ObjectPortlet.assertEntryNotPresent(entry = "Build Incredible Digital Experiences with Liferay DXP");
+		}
+
+		task ("and When deactivating the custom validation") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 192646");
+
+			ObjectAdmin.goToValidationsTab();
+
+			ObjectCustomValidation.gotoCustomValidation(validationName = "Custom Validation");
+
+			ObjectCustomValidation.desactiveValidation();
+
+			PortletEntry.save();
+		}
+
+		task ("and When adding the same incorrect object entry before") {
```

Comment:
and When the user deactivates the custom validation and attempts to create the same invalid object entry